incubator/zookeeper/templates/ss.yaml

```diff
@@ -142,11 +140,7 @@ spec:
   - metadata:
       name: datadir
       annotations:
-        {{- if .Values.StorageClass}}
-        volume.beta.kubernetes.io/storage-class: {{.Values.StorageClass | quote}}
-        {{- else}}
-        volume.alpha.kubernetes.io/storage-class: "default"
-        {{- end}} 
+        volume.beta.kubernetes.io/storage-class: {{default "standard" .Values.StorageClass | quote}}
```

Comment:
After re-reading up on https://github.com/kubernetes/charts/issues/116#issuecomment-262414987, I think we should leave out the annotation by default and use it if .Values.StorageClass is defined. Leaving out the annotation ensures the default storage class is used on 1.6, which is more portable given `standard` is not always the name of the default storageclass (https://github.com/kubernetes/kubernetes/commit/504651479781a8878c4ec00e195e2e9c5546c913)